src/dataframe/dataframe.jl

```diff
@@ -83,10 +83,10 @@ size(df1)
 
 """
 mutable struct DataFrame <: AbstractDataFrame
-    columns::Vector
+    columns::Vector{AbstractVector}
     colindex::Index
 
-    function DataFrame(columns::Vector{Any}, colindex::Index)
+    function DataFrame(columns::Vector, colindex::Index)
         if length(columns) == length(colindex) == 0
             return new(Vector{Any}(undef, 0), Index())
```

Comment:
This one needs to be changed (`AbstractVector[]` would be shorter BTW).

Reply:
see comment below